app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/NavigationUtils.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
+
+import androidx.activity.OnBackPressedCallback
+import androidx.navigation.NavController
+
+class NavigationUtils {
+    companion object {
```

Comment:
```suggestion
object NavigationUtils {
```

We will only ever have the object, just like `TestUtils`